dags/glam_subdags/extract.py

```diff
@@ -160,3 +160,63 @@ def extract_user_counts(
     etl_query >> gcs_delete >> bq2gcs
 
     return dag
+
+def extract_sample_counts(
```

Comment:
Since `extract_sample_counts()` is very similar to `extract_user_counts()`, can we refactor the function so that the same one can be used and just pass in the required params for each call?